app/posts/apply-for-teacher-training/2020-01-21-suitability-to-work-with-children.md

```diff
@@ -41,11 +45,8 @@ Any 'disclosure' would be superseded by the DBS so it is rather redundant.
 
 ## User needs
 
-As a provider I need to know if a candidate has a history that would affect their application, so that I can either discuss and assist them in their journey, or reject them, if appropriate.
-
-As a candidate with a checkered history I need an opportunity to be honest about my past, so that I have a better chance of becoming a teacher.
-
-As a candidate with a checkered history I need to know what a DBS check is, and what it will say about me, so that I can make a decision about what to disclose beforehand.
+{% from "user-needs/macro.njk" import appUserNeeds %}
+{{ appUserNeeds({ items: collections['user-need'] | slugs(tags)}) }}
 
```

Comment:
I think this can be simplified further to:
```{% from "user-needs/macro.njk" import appUserNeeds with context %}
{{ appUserNeeds }}```

Then within the user needs macro you will have the context of the current post, so you can hide `items: collections['user-need'] | slugs(tags)` away in the macro.